tensorflow/python/ops/rnn.py

```diff
@@ -580,7 +580,9 @@ def dynamic_rnn(cell,
         tuple of Tensors each with dimensions `[batch_size, ...]`.
     sequence_length: (optional) An int32/int64 vector sized `[batch_size]`. Used
       to copy-through state and zero-out outputs when past a batch element's
-      sequence length.  So it's more for performance than correctness.
+      sequence length.  This argument enables this API to extract
```

Comment:
Change to:

This parameter enables users to extract the last valid state and properly padded outputs, so it is provided for correctness.

Reply:
Done in ca644c9